code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/AlekseevSergeiTest.java

```diff
@@ -0,0 +1,78 @@
+package ru.unn.agile.ComplexNumber.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class AlekseevSergeiTest {
+
+    @Test
+    public void areAddingZeroDoesntChangeNumber() {
+        ComplexNumber z = new ComplexNumber(1.1, 2.3);
+        ComplexNumber zero = new ComplexNumber(0.0, 0.0);
+
+        ComplexNumber sum = z.add(zero);
+
+        assertTrue(sum.equals(z));
```

Comment:
it’s better not to equals in assert:

    public void areAddingZeroDoesntChangeNumber() {
        //Arrange  
        ComplexNumber z = new ComplexNumber(1.1, 2.3);
        ComplexNumber zero = new ComplexNumber(0.0, 0.0);

        //Act
        ComplexNumber sum = z.add(zero);

        //Assert
        assertEquals(sum, z);